deep-learning/_posts/2017-02-10-vnet.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: review
+title: V-Net:Fully Convolutional Neural Networks for Volumetric Medical Image Segmentation
+tags: deep-learning segmentation medical
+cite:
+  - V-Net
+  - http://campar.in.tum.de/pub/milletari2016Vnet/milletari2016Vnet.pdf
+---
+
+<style>
+  main.page-content p {
+    text-align: justify; /* helps the reading flow */
+  }
+</style>
+
+In this paper they address the segmentation of the prostate challenge [PROMISE2012](https://promise12.grand-challenge.org/){:target="_blank"}.
```

Comment:
Not sure I understand.  The text should be something like

New MRI segmentation method of the prostate based on a 3D CNN similar to the U-Net.  Results reported on the prostate challenge [PROMISE2012](https://promise12.grand-challenge.org